collision-map-update/CollisionMapDumper.java

```diff
@@ -227,148 +227,82 @@ private void addCollisions(FlagMap flagMap, Region region)
 						{
 							Z = z != tileZ ? z : loc.getPosition().getZ();
 
-							if (object.getMapSceneID() != -1)
+							boolean door = object.getWallOrDoor() != 0 && (object.getName().toLowerCase().contains("door") || object.getName().toLowerCase().contains("gate"));
```

Comment:
I don't really like this hardcode approach with `contains("door")` and `contains("gate")`. I'm sure there are plenty of doors or gates without `door` or `gate` in the name. It might be possible to use something like the red colour to detect them, but it looks like stiles would disappear (which is bad). Do you have some image examples of where `contains("door")` or `contains("gate")` improves over the previous approach? Also, many of the exclusions in the `Exclusion` enum exist because of the current gate/door detection, so some of the exclusions might be wrong now (or creating false positives for your new door/gate detection). 
![image](https://github.com/user-attachments/assets/b3e771ae-23db-4ab3-a572-89bc42732cfd)